R/registry.R

```diff
@@ -57,20 +57,24 @@ get_status <- function(entry) {
     status <- guess_status(entry)
   }
   status <- gsub("http(s)?\\:\\/\\/www\\.repostatus\\.org\\/\\#",
-    "https://www.repostatus.org#", status)
+                 "https://www.repostatus.org#", status)
   return(status)
 }
 
 guess_status <- function(entry) {
```

Comment:
adding type was an opportunity to clean up `guess_status()` (early returns :-) )